@ecomplus/storefront-renderer/functions/index.js

```diff
@@ -49,6 +62,9 @@ exports.ssr = (req, res, getCacheControl) => {
   const fallback = () => {
     if (url.slice(-1) === '/') {
       redirect(url.slice(0, -1))
+    } else if (url.startsWith('/reverse-proxy/')) {
+      const { data, status, headers } = proxy(url)
+      res.writeHead(status, headers).send(data)
```

Comment:
```suggestion
      proxy(url).then(({ data, status, headers }) => {
        res.writeHead(status, headers).send(data)
      })
```

A função é assíncrona (request), retorna uma promise...

Reply:
Uma coisa que não entendo, eu tinha colocado async lá no início da função, porque sabia que era uma promise e tinha colocado um await, mas falava no vscode que não tinha efeito nenhum, por isso retirei. Agora é sempre usar then, porque sei lá que as vezes acontece isso com async await